packages/Card/Card.jsx

```diff
@@ -1,20 +1,35 @@
 import React from 'react'
 import PropTypes from 'prop-types'
+import styled from 'styled-components'
 
 import Box from '@tds/core-box'
+import { colorWhite, colorWhiteLilac, colorAthensGrey } from '@tds/core-colours'
+import { borders } from '@tds/shared-styles'
 import safeRest from '../../shared/utils/safeRest'
 
-import styles from './Card.modules.scss'
-
+const StyledCard = styled(({ variant, ...rest }) => <Box {...rest} />)(({ variant }) => ({
+  ...borders.none,
+  ...borders.rounded,
+  ...(variant === 'white' && {
+    boxShadow: '0 0 16px 0 rgba(213, 213, 213, 0.5)',
```

Comment:
Is there an opportunity to move this to `@tds/shared-styles`?

Reply:
I don't think so, this is only used in Card as far as I'm aware.